rllib/agents/ddpg/td3.py

```diff
@@ -51,15 +50,16 @@
         "target_network_update_freq": 0,
         "num_workers": 0,
         "num_gpus_per_worker": 0,
-        "worker_side_prioritization": False,
```

Comment:
Cool, that we are moving this into the replay buffer config as well. Looks much cleaner now.